patches/0062-Allow-DHCP-Broadcast-packets.patch

```diff
@@ -0,0 +1,74 @@
+From 52c1cff91bab2da916816783f53dc5ee47a9ee13 Mon Sep 17 00:00:00 2001
```

Comment:
Why two commits for the same bug fix?

Reply:
WIFI5 and WIFI6 kernel code is different since they are running on different linux kernel versions.

Reply:
Yes, but the two patches can be in the same commit. push it on a branch from wifi5, and pull the same branch and add the wifi6 code.

Reply:
Yes, that can be doable. But for time being I'm keeping them separate for now.